CardScan/Classes/NMS.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  NMS.swift
+//  CardScan
+//
+//  Created by Zain on 8/6/19.
+//
+
+import Foundation
+import os.log
+struct NMS{
+    static func hardNMS(subsetBoxes: [[Float]], probs: [Float], iouThreshold: Float, topK: Int, candidateSize: Int) -> [Int] {
+        /** In this project we implement HARD NMS and NOT Soft NMS
+         * I highly recommend checkout SOFT NMS Implementation of Facebook Detectron Framework
+         *
+         *  Args:
+         *  box_scores (N, 5): boxes in corner-form and probabilities.
+         *  iou_threshold: intersection over union threshold.
+         *  top_k: keep top_k results. If k <= 0, keep all the results.
+         *  candidate_size: only consider the candidates with the highest scores.
+         *
+         *  Returns:
+         *  picked: a list of indexes of the kept boxes
+         */
+        
+        
+        let sorted = probs.enumerated().sorted(by: {$0.element > $1.element})
```

Comment:
the sorted part I think can just be `sorted { $0.element > $1.element }` no need for a label or passing it as an argument.